source/community/reactnative/src/components/actions/action-button.js

```diff
@@ -38,7 +38,7 @@ export class ActionButton extends React.Component {
 		this.title = Constants.EmptyString;
 		this.altText = Constants.EmptyString;
 		this.type = Constants.EmptyString;
-		this.iconUrl = Constants.EmptyString;
+		this.url = Constants.EmptyString;
```

Comment:
It is iconUrl for action elements. Please refer to the schema - https://adaptivecards.io/explorer/Action.OpenUrl.html

Reply:
Implemented as per the changes requested at https://github.com/microsoft/AdaptiveCards/issues/3053